# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/crashtests.list
# Commit: ce9f99b79f12
# Full Hash: ce9f99b79f12aae2cf231004197b92b3b372e0bc
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2019-07-17 09:36:40
# Description:
#   Bug 1566310 - Nullcheck mWindowGlobalChild prior to notifying it of beforeunload listeners, r=nika
#   
#   Missed in the 1543251 refactor.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D38223
# ==============================================================================

diff -r 663f5d1ba3f3 -r ce9f99b79f12 dom/base/crashtests/crashtests.list
--- a/dom/base/crashtests/crashtests.list	Tue Jul 16 23:40:47 2019 +0300
+++ b/dom/base/crashtests/crashtests.list	Tue Jul 16 18:35:25 2019 +0000
@@ -246,6 +246,7 @@
 load 1516289.html
 load 1516560.html
 load 1528675.html
+load 1566310.html
 load structured_clone_container_throws.html
 load xhr_empty_datauri.html
 load xhr_html_nullresponse.html